frontend/pages/features.tsx

```diff
@@ -24,24 +25,28 @@ export default function Page() {
 
 	return (
 		<main className="flex flex-1">
-			<Head>
-				<title>Kioku</title>
-				<meta name="description" content="Kioku" />
-				<link rel="icon" href="/favicon.ico" />
-				<link
-					rel="alternate"
-					hrefLang="en"
-					href="https://app.kioku.dev/features"
-				/>
-				<link
-					rel="alternate"
-					hrefLang="de"
-					href="https://app.kioku.dev/de/features"
-				/>
-			</Head>
+			<NextSeo
+				title={_(msg`Kioku | Features of the Kioku platform`)}
+				description={_(
+					msg`Discover the features of our free flashcard application. Import decks from Anki, create groups, learn together with friends and compare your statistics. Now on all devices!`
+				)}
+				languageAlternates={[
+					{ hrefLang: "en", href: "https://app.kioku.dev/features/" },
```

Comment:
```suggestion
					{ hrefLang: "en", href: "https://app.kioku.dev/features" },
```
is the last `/` required? also in `openGraph`

Reply:
no, I removed them